internal/tpm/tpm.go

```diff
@@ -71,6 +71,13 @@ func (t *TPM) Close() {
 	}
 }
 
+func (t *TPM) GetTpmVendorInfo() ([]byte, error) {
+	if t == nil || t.channel == nil {
+		return []byte{}, nil
```

Comment:
this isn't an error? should we log this at least it feels pretty unexpected

Reply:
Hmm. Initially I thought it might be fine, but I think you are right.